src/interface/lineplot.jl

```diff
@@ -101,8 +101,7 @@ function lineplot(
         color::UserColorType = :auto,
         name = "",
         kw...)
-    idx = map(!isnan, x)
-    idx .&= map(!isnan, y)
+    idx = map(isfinite, x) .& map(isfinite, y)
```

Comment:
This doesn't matter much but I'm sensitive to unnecessary memory allocations :)

```suggestion
    idx = map(x, y) do i,j
        isfinite(i) && isfinite(j)
    end
```

Reply:
You're absolutely right !